src/vs/platform/backup/electron-main/backupMainService.ts

```diff
@@ -51,11 +51,6 @@ export class BackupMainService implements IBackupMainService {
 	}
 
 	public registerWindowForBackups(windowId: number, isEmptyWorkspace: boolean, backupFolder?: string, workspacePath?: string): void {
```

Comment:
Hm yeah maybe. We can leave it there for now. The problem is that the main side only ever has one environment service and so a second window would not answer this check correctly.